src/ca11y/lib/parser.js

```diff
@@ -70,13 +73,6 @@ export default function parser(string, format, delimiter) {
     return formats
   }, {})
 
-  const { yyyy, mm, dd } = normalized
-  const valid = yyyy && mm && dd
-
-  const date = new Date()
-  const offset = date.getTimezoneOffset() * 60 * 1000
-  const parsed = Date.parse(`${yyyy}-${mm}-${dd}`)
-  date.setTime(parsed + offset)
-
-  return { date, valid }
```

Comment:
@solomonhawk I killed the validity stuff here... if you enter a number that's too high (like 32 for a month value), `Date` just does the best it can and would increment the month, and set the day to 1. Users see the input change from an invalid to a valid date, rather than leaving an invalid date in the input and defaulting the picker to the current date.